src/Magnum/Trade/PhongMaterialData.h

```diff
@@ -299,11 +375,15 @@ class MAGNUM_TRADE_EXPORT PhongMaterialData: public AbstractMaterialData {
            and thus better noticeable */
         Color4 _ambientColor;
         UnsignedInt _ambientTexture{~UnsignedInt{}};
+        UnsignedInt _ambientTextureLayer{0};
         Color4 _diffuseColor;
         UnsignedInt _diffuseTexture{~UnsignedInt{}};
+        UnsignedInt _diffuseTextureLayer{0};
         Color4 _specularColor;
         UnsignedInt _specularTexture{~UnsignedInt{}};
+        UnsignedInt _specularTextureLayer{0};
         UnsignedInt _normalTexture{~UnsignedInt{}};
+        UnsignedInt _normalTextureLayer{0};
```

Comment:
Would do the same as for textures here, i.e.

```suggestion
        UnsignedInt _normalTextureLayer{~UnsignedInt{}};
```

so we catch accidental access of invalid values better.

Reply:
I assume this would be an uninitialized PhongMaterialData? In Importers where this is not set explicitly, it should be 0 anyway... unless the flag suggested below should be preventing the access of these entirely?

Reply:
> the flag suggested below should be preventing the access of these entirely

Yes, basically the same behavior as for `_normalTexture` etc.